lib/simp/beaker_helpers.rb

```diff
@@ -936,4 +944,40 @@ def install_puppet
 
     run_puppet_install_helper(install_info[:puppet_install_type], install_info[:puppet_install_version])
   end
+
+  def install_simp_deps_repo(sut, version = '6_X_Dependencies', gpgcheck = '1')
```

Comment:
Please add the ability to add the SIMP repo and the SIMP dependency repo.  Please see https://github.com/simp/simp-core/blob/master/spec/acceptance/helpers/repo_helper.rb#L51-L111

Reply:
So, while I completely get what we're going for here, we are probably going to be moving these in the near future and adding this to `beaker_helpers` is going to require us to start pinning different versions of `beaker_helpers` in different modules based on the target.

I don't know that we actually will want to use this function as time progresses. I definitely agree that it's irritating to update everything but it's more explicit in the tests themselves and we don't actually install the deps in all of our tests (or most of them IIRC).

Reply:
I disagree.  As it stands, every time the repos change (new changes are in progress), we have to hunt down all cases of acceptance tests that set up the repos either with helper methods or repo defs in the node sets.  This is painful to update.  I think we should have some method to install the repo file for the one/subset/all of the repos repos and use it everywhere.

Reply:
Fair enough. In that case, I guess we'll need to be able to pass this a list of repos to disable out of the usual stack and that list will change based on what this function points to at the time.

Making it a `disable` list instead of an `enable` list should ensure that the fewest number of items is affected when things move around.

Checking to see if what is passed matches the internal list in the function and failing would make sure that we update in a timely manner.